packages/suite/src/hooks/wallet/sign-verify/useSignVerifyForm.ts

```diff
@@ -33,15 +33,11 @@ const DEFAULT_VALUES: SignVerifyFields = {
 };
 
 const signVerifySchema = yup.object<ValidationSchema<Omit<SignVerifyFields, 'isElectrum'>>>({
-    message: yup
-        .string()
-        .max(MAX_LENGTH_MESSAGE, 'TR_TOO_LONG')
-        .when('hex', {
-            is: true,
-            then: schema => schema.isHex(),
-            otherwise: schema => schema.isAscii(),
-        })
-        .required(),
+    message: yup.string().max(MAX_LENGTH_MESSAGE, 'TR_TOO_LONG').required().when('hex', {
```

Comment:
what was problem? why previous version did not work?

Reply:
Actually, the problem was in the arrow functions for custom validators ;D `this` in those is important after all.